schemas/HPXMLBaseElements.xsd

```diff
@@ -6230,6 +6395,56 @@
 		</xs:sequence>
 		<xs:attribute name="dataSource" type="DataSource"/>
 	</xs:complexType>
+	<xs:complexType name="VehicleBattery">
```

Comment:
Seems like most of the elements here are copies of the elements in the `Battery` type, right? Ideally we would have a base `Battery` type with common elements, and then `VehicleBattery` and `HomeBattery` elements would use the base type and add any additional/custom properties. That way we don't accidentally add new elements in the future to vehicles but forget to add them to home batteries. But not critical to change...

Reply:
Would that result in breaking changes though? Where the home battery elements are now split between the common `Battery` element and the home battery level?

Reply:
It wouldn't be breaking, no. It changes the underlying infrastructure but would not change the home battery elements that are exposed to the user.

Reply:
I just realized that it could change the order of elements for the `HomeBattery` though, which would be a breaking change. If that's true, then I suggest not doing this. Nobody likes a breaking change, we should only do it if absolutely necessary. Sorry for the hassle.